CMakeLists.txt

```diff
@@ -87,13 +66,15 @@ if(APPLE)
     endif()
     message(STATUS "Using MoltenVK repo location at ${MOLTENVK_REPO_ROOT}")
 endif()
-message(STATUS "Using find_package to locate Vulkan")
-find_package(Vulkan)
+
 find_package(VulkanHeaders REQUIRED CONFIG)
-get_filename_component(Vulkan_LIBRARY_DIR ${Vulkan_LIBRARY} DIRECTORY)
-message(STATUS "Vulkan FOUND = ${Vulkan_FOUND}")
-message(STATUS "Vulkan Lib Dir = ${Vulkan_LIBRARY_DIR}")
-message(STATUS "Vulkan Lib = ${Vulkan_LIBRARY}")
+
+# Find and create the Vulkan::Vulkan library
```

Comment:
This comment isn't very helpful.

Reply:
What I mean is Vulkan::Vulkan only makes sense as a target name if you are aware of the existing Vulkan support built into to CMake

Reply:
I think something like vulkan_loader is much more clear. This one is very much an opinion though.

But this comment really doesn't say anything that the code below doesn't.

Reply:
The only hiccup I have changing Vulkan::Vulkan to vulkan_loader is that it *also* includes the headers, which isn't obvious from the name vulkan_loader. Happy to change Vulkan::Vulkan to something more descriptive, it is confusing to use the same name as the existing CMake vulkan support. I can totally see myself seeing Vulkan::Vulkan in this repo and coming to the wrong conclusion.

Reply:
Maybe I'm just overthinking it. Looks good to me overall.